routes/article.js

```diff
@@ -99,10 +93,10 @@ router.get('/new', (req, res) => {
 
     getNextArticles(findParams, sortParams)
         .then(articles => res.send(articles))
-        .catch((err) => {
+        .catch(err => {
```

Comment:
Or you can make it a one line express like this:
```javascript
.catch(err => (
  res.status(500).send({
    err: `Newest articles weren't found: ${err}`,
    givens: req.query
  }))
```

Style from https://github.com/airbnb/javascript#arrows--paren-wrap